packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -56,7 +56,8 @@ class MyApp extends StatelessWidget {
         // or simply save your changes to "hot reload" in a Flutter IDE).
```

Comment:
we should update the comment to be accurate :-)

Reply:
Well, it's usually correct, since that is the default (to hot reload on save). I was making it more accurate by adding the "or press the hot reload button", but it's a minor thing. Just saving will work for most (probably 99%) of new users.

Reply:
I would argue for prioritizing the hot reload instruction for IDEs over that for CLI here, since our [Test drive doc](https://docs.flutter.dev/get-started/test-drive) is very much IDE-based now. Referencing the "r" and "flutter run" commands could be confusing for new users following the test drive instructions.

Reply:
Okay, I reordered the instructions here to give the IDE version first.